app/controllers/v0/intent_to_files_controller.rb

```diff
@@ -16,7 +16,7 @@ class IntentToFilesController < ApplicationController
       :index,
       :submit,
       additional_class_logs: {
-        action: 'load Intent To File for 526 form flow'
+        action: 'Intent To File transction'
```

Comment:
nit: typo in `transaction`

Reply:
ty!  I'm a terrible speller, please always call em if you see em